.github/workflows/aro-hcp-dev-what-if.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: ARO HCP Integrated DEV Bicep What-If
+env:
+  DEPLOY_ENV: dev
+  DEPLOYMENT_NAME_SUFFIX: "-dev-${GITHUB_RUN_ID}"
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'dev-infrastructure/**/*.bicep'
+      - 'dev-infrastructure/**/*.bicepparam'
+      - 'dev-infrastructure/Makefile'
+      - '.github/workflows/aro-hcp-dev-bicep-what-if.yml'
```

Comment:
Do we need to add config in here as well if it changes to run?

Reply:
good catch. fixed